src/features/cosoul/CoSoulButton.tsx

```diff
@@ -80,17 +78,18 @@ export const CoSoulButton = ({ onReveal }: { onReveal(): void }) => {
             another L2 (eg. Base, Arbitrum).
           </Text>
           <Flex column css={{ mt: '$sm' }}>
-            {/* {defaultAvailableChains.includes(chainId as ChainId) &&             (!IN_PREVIEW || (IN_PREVIEW && isFeatureEnabled('test_decent'))) ? (
+            {defaultAvailableChains.includes(chainId as ChainId) &&
+            (!IN_PREVIEW || (IN_PREVIEW && isFeatureEnabled('test_decent'))) ? (
```

Comment:
Since Decent is live, can we just remove this and all uses of `test_decent` and just show Decent everywhere? @sshmm

Reply:
@crabsinger this feature flag is there to allow testing the convertion to optimism instead of optimism sepolia on staging and to also to prevent using actual tokens by mistake while testing

Reply:
ah, makes sense. great.